app/src/ui/lib/update-store.ts

```diff
@@ -189,6 +189,11 @@ class UpdateStore {
     // Send the GUID to the update server for staggered release support
     url.searchParams.set('guid', await getRendererGUID())
 
+    if (!staggered) {
+      // If omitted, staggered releases will be enabled by default
+      url.searchParams.set('staggered', '0')
```

Comment:
Not a fan of using parameters that by default, when omitted, are `true`, but I also didn't want to use negative logic (like `nonStaggered=1`)

In any case, open to suggestions 😄

Reply:
or `skipGuidCheck` ... having parameter to skip things is a pattern I have seen elsewhere

Reply:
`forced` was my first option but it's like… "what's it forcing exactly?" (which is also why the `(Forced)` suffix in the button doesn't convince me 😂 )

And verb+`stagger` didn't sound very well (grammatically) in my head 😂 `skipGuidCheck` could be closer to what we want, but might be confusing since there is a special case where no GUID is sent.

Maybe I shouldn't overcomplicate things and just use `forced` 🤣

Reply:
Yeah. I think (Forced) on the button has context enough that it doesn't feel as funny to me and can't think user friendly/simple alternative. But.. .passing just the word "forced" as the parameter doesn't seem to me like it has enough context.. I lean more towards `skipGuidCheck` for the param name. It doesn't bother me being a special case there is no GUID sent since that is not the expected and if there isn't a guid it makes sense that skipGuidCheck doesn't apply in that special scenario.

Reply:
I think your initial inclination about the word choice here is worth listening to. Forced is a word with mostly negative connotations. It's hard to find a drop in replacement so I'm wondering if we instead could change the button label entirely. Perhaps to something like "Ensure latest version" coupled with a tooltip on hover which explains what's going to happen and how it differs from the regular button?

<img src="https://user-images.githubusercontent.com/634063/185145261-81eb6af0-e768-44e6-8153-b9afa795b008.gif" width="484" />

Reply:
I like "Ensure Latest Version" -> agree that is much more user friendly then "forced". The tooltip is great too.